_projects/meta_review.md

```diff
@@ -0,0 +1,94 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks and provides meaningful analysis of reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and JavaScript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The meta-review project is about building a meta-review track system,
```

Comment:
track -> tracking, or maybe just remove 'track' here as the word track is used later in the sentence.